charts/consul/values.yaml

```diff
@@ -1556,8 +1556,8 @@ connectInject:
   # This setting can be safely disabled by setting to "Ignore".
   failurePolicy: "Fail"
 
-  # Selector for restricting the webhook to only
-  # specific namespaces. This should be set to a multiline string.
+  # Selector for restricting the webhook to only specific namespaces. 
+  # Use with `connectInject.default` to enable/disable connectInject by default via the selector. This should be set to a multiline string.
```

Comment:
This is still a bit confusing. Maybe we can spell it out directly? "Use with `connectInject.default: true` to automatically inject all pods in namespaces that match the selector"

Reply:
That sounds good